package.json

```diff
@@ -3,9 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/core": "^10.0.28",
+    "@emotion/styled": "^10.0.27",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "emotion": "^10.0.27",
```

Comment:
I think you do not need this dependency as you already have @emotion core and styled, which are needed for React

Reply:
yepp thanks, I forgot to uninstall it! :)